youtube_dl/extractor/blick.py

```diff
@@ -0,0 +1,138 @@
+            'thumbnail': 'http://f.blick.ch/img/incoming/crop5066860/5146024130-csquare-w300-h300/Bildschirmfoto-2016-05-23-um-14.jpg',
+            'description': 'Da ist Papa Heinz mächtig stolz. Seine Tochter Alessandra Günthardt ist für einen schwedischen Musik-Preis unter den drei Nominierten. Die Abstimmung läuft noch bis 7. Juni.'
+        }
+    }, {
+        'url': 'http://www.blick.ch/sport/fussball/superleague/totomat-fehler-in-sion-fcz-buff-stinksauer-wegen-falschem-lugano-resultat-id5063421.html',
+        'info_dict': {
+            'id': '5063421',
+            'ext': 'mp4',
+            'title': 'totomat-fehler-in-sion-fcz-buff-stinksauer-wegen-falschem-lugano-resultat',
+            'thumbnail': 'http://f.blick.ch/img/incoming/crop5063475/820602933-csquare-w300-h300/Bildschirmfoto-2016-05-22-um-19.jpg',
+            'description': 'Der FC Zürich bleibt das Schlusslicht der Raiffeisen Super League. Einen dicken Hals bekommen Buff und Co. aber wegen einer falschen Resultatanzeige aus dem Ländle.',
+        }
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        video_id = mobj.group('id')
+        webpage = self._download_webpage(url, video_id)
+
+        found_videos = []
+        regex_og = self._og_regexes('video')
+        regex_ogs = self._og_regexes('video:secure_url')
+        video_og = self._html_search_regex(regex_og, webpage, name=None, default=None, fatal=False)
+        video_ogs = self._html_search_regex(regex_ogs, webpage, name=None, default=None, fatal=False)
+        video_meta = self._html_search_meta('contentURL', webpage, fatal=False, default=None)
+        for elem in [video_og, video_ogs, video_meta]:
+            if elem:
+                found_videos.append(elem)
+
+        video_url = ''
+        for video in found_videos:
+            if re.match(r'.*detect\.mp4', video):
+                ind = video.rfind('/')
+                video_url = video[:ind + 1]
+                video_url += 'index.m3u8'
+                break
+            elif re.match(r'.*\.m3u8', video):
+                video_url = video
+                break
+
+        if not video_url:
+            return []
+
+        video_title = str(url)
+        b_ind = video_title.rfind('/') + 1
+        e_ind = video_title.rfind('-id')
+        video_title = video_title[b_ind:e_ind]
+
+        video_description = self._og_search_description(webpage)
+        thumbnail = self._og_search_thumbnail(webpage)
+
+        entry_info_dict = {
+            'id': video_id,
+            'title': video_title,
+            'description': video_description,
+            'thumbnail': thumbnail,
+            'duration': None,
+        }
+        entry_info_dict['formats'] = self._extract_m3u8_formats(
+            video_url,
+            video_id,
+            ext='mp4',
+            entry_protocol='m3u8_native')
+
+        if entry_info_dict.get('formats'):
```

Comment:
This check does not make any sense. If there are no formats extraction should stop immediately.